lib/renderer/init.js

```diff
@@ -70,11 +70,13 @@ if (window.location.protocol === 'chrome-devtools:') {
   // Override default web functions.
   require('@electron/internal/renderer/window-setup')(ipcRenderer, guestInstanceId, openerId, isHiddenPage, usesNativeWindowOpen)
 
-  // Inject content scripts.
-  require('@electron/internal/renderer/content-scripts-injector')
+  // Inject content scripts in the main frame only
+  if (process.isMainFrame) {
+    require('@electron/internal/renderer/content-scripts-injector')
+  }
 
   // Load webview tag implementation.
-  if (webviewTag && guestInstanceId == null) {
+  if (webviewTag && guestInstanceId == null && process.isMainFrame) {
```

Comment:
it's intentional that `<webview>`s aren't allowed in subframes?

Reply:
Yes